Doc/library/dis.rst

```diff
@@ -1059,9 +1059,8 @@ All of the following opcodes use their arguments.
 .. opcode:: LOAD_CLOSURE (i)
 
    Pushes a reference to the cell contained in slot *i* of the cell and free
-   variable storage.  The name of the variable is ``co_cellvars[i]`` if *i* is
-   less than the length of *co_cellvars*.  Otherwise it is ``co_freevars[i -
-   len(co_cellvars)]``.
+   variable storage.  The name of the variable is
+   ``co_fastlocalnames[i + len(co_varnames)]``.
```

Comment:
Um, I thought we were switching to consistent numbering? Shouldn't it just be `co_fastlocalnames[i]`? Or is that the next PR?

Reply:
the next PR